src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="block">
+        <legend class="block__title">
+          Personal information
+        </legend>
+
+        <label class="block__input">
```

Comment:
cool!

Reply:
spaces in textarea are there because of tabs in code before it. i don`t why the tabs affects the content in textarea because this tabs are for code style. when i put the closing tag right on the row with the opening tag the problem disappears but linter is against this solution and tells me to put closing tag on the next line and again the tabs are making content in textarea, i don't know how to solve this problem
screen (https://prnt.sc/-61rbdRup6T-)

Reply:
i also was watching some solutions from my mates which were approved and i've found one where the textarea has the same problem as mine but the solution was approved so i can't understand is it my mistake or it's a problem in browser or vscode :)